marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/entitySearch/exportSearchTest.sjs

```diff
@@ -1,18 +1,29 @@
 const search = require('/MarkLogic/appservices/search/search');
 const test = require("/test/test-helper.xqy");
 const hubTest = require("/test/data-hub-test-helper.sjs");
-const op = require('/MarkLogic/optic');
 
-const queryOptions = '<options xmlns="http://marklogic.com/appservices/search"/>';
+const queryOptions = `<options xmlns="http://marklogic.com/appservices/search"/>`;
 
-function invokeOpticPlanService(schemaName, viewName, limit, structuredQuery, searchText, sortOrder, columns, retries=0) {
-    try {
-        // Need to clear the modules cache or unrelated error could be thrown. Investigating to file a bugtrack.
-        if (retries === 0) {
-            xdmp.moduleCacheClear();
+// This is a simple CSV parser intended for simplifying tests only.
+function basicCsvParser(csv) {
+    const lines = csv.split('\n').map((line) => fn.normalizeSpace(line)).filter((line) => line !== '');
+    const results = [];
+    const headers = lines[0].split(',').map((header) => fn.normalizeSpace(header));
+    for (const line of lines.slice(1)) {
+        const obj = {};
+        let properties = line.split(',').map((val) => fn.normalizeSpace(val));
+        for (let i = 0; i < properties.length; i++) {
+            obj[headers[i]] = properties[i];
         }
+        results.push(obj);
+    }
+    return Sequence.from(results);
+}
+
+function invokeExportSearchService(schemaName, viewName, limit, structuredQuery, searchText, sortOrder, columns, retries=0) {
+    try {
         return fn.head(xdmp.invoke(
-            "/data-hub/5/data-services/entitySearch/getOpticPlan.sjs",
+            "/data-hub/5/data-services/entitySearch/exportSearch.sjs",
```

Comment:
Needs to be updated to the new name `exportSearchAsCSV`

Reply:
renamed